extra_model/_aspects.py

```diff
@@ -65,12 +62,10 @@ def adjective_list(tokens):
     for child in tokens:
         if child.dep == amod:
             adjectives.append(child.text)
-            for (
-                grandchild
-            ) in (
-                child.children
-            ):  # find both X and Y in patterns of the form "the X and Y product"
-                if grandchild.dep_ == "conj":
+            for grandchild in child.children:
+                # find both X and Y in patterns of the form "the X and Y product"
+                if grandchild.dep_ == "conj" and not grandchild.is_space:
+                    # grandchild.is_space is handling double space cases
                     adjectives.append(grandchild.text)
```

Comment:
might make sense to move this to a generic helper function as this is basically duplicate code from line 43-49

Reply:
Fine by me, but we should at least keep track of this and implement eventually. Do you have a place to keep track of such things?